### ISSUE TITLE
Remove the "Sign Up" button from navbar

### ISSUE THREAD

**Issue by winston-yallow** [godotengine/discourse-theme#15]
This is a regression due to the last discourse update. We only need one button for log in and registration, not two.

![image](https://github.com/godotengine/discourse-theme/assets/44872771/ccded27b-7683-4748-8abc-b5153c0d7e83)

**Comment by coppolaemilio** [comment]
I guess we need to modify this from our theme, right?

**Comment by winston-yallow** [comment]
Yes, should be easy to do. We already had that, but I think the button class changed with the last update.

Code is here on line 203:
https://github.com/godotengine/discourse-theme/blob/main/common/common.scss#L203
